### DISCUSSION TITLE
Expand when zooming in a collapsed row (of an Outliner list)

### DISCUSSION THREAD

**Discussion by Tamriel** [vslinko/obsidian-zoom#116] [Ideas]
When combined with Outliner, zooming in on a collapsed row does not help much since you need to expand it. I wish this plugin would do this automatically, and when zooming out to collapse the row again.
( Just like Logseq does this )

**Comment by haukot** [comment]
Have the same problem, not sure maybe it is a bug? Cause autoexpand is a main feature why I like zoom in outliners. Otherwise either you have to fold/unfold each time manually, or the main list will be pretty big with all insides expanded
